core-db/src/main/java/io/novafoundation/nova/core_db/model/chain/ChainLocal.kt

```diff
@@ -22,6 +22,7 @@ data class ChainLocal(
     @ColumnInfo(defaultValue = "1")
     val hasSubstrateRuntime: Boolean,
     val hasCrowdloans: Boolean,
+    val swapSupporting: Boolean,
```

Comment:
We should take into account that there will be multiple swap types in the future so it is better to make it a nullable enum with a single value ASSET_CONVERSION